web/pandas/versions.json

```
@@ -1,30 +1,37 @@
 [
     {
         "name": "dev",
-        "version": "docs/dev"
+        "version": "dev",
+        "url": "https://pandas.pydata.org/docs/dev/index.html"
```

```suggestion
        "url": "/docs/dev/"
```
Those are the links I was referring to. I think this should still work,without the `index.html` also if I'm not missing anything. And I think by not having the domain, the website/docs are more portable (if you set up things locally or in a dev server for tests for example, navigation wouldn't take you to the prod server, and would stay in the same website you started).

But no big deal, happy to merge this as is too.

---

Ah sorry, I changed them locally but I must have edited a non git file. 

This is what I wanted to use too, will push an update soon

---

Updated